chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -261,6 +263,28 @@ impl PeerManagerActor {
             client,
             whitelist_nodes,
         ));
+        arbiter.spawn({
+            let arbiter = arbiter.clone();
+            let state = state.clone();
+            let clock = clock.clone();
+            async move {
+                if let Some(cfg) = state.config.tier1.clone() {
+                    // Connect to TIER1 proxies and broadcast the list those connections periodically.
+                    arbiter.spawn({
+                        let clock = clock.clone();
+                        let state = state.clone();
+                        let mut interval =
+                            time::Interval::new(clock.now(), cfg.advertise_proxies_interval);
+                        async move {
+                            loop {
+                                interval.tick(&clock).await;
+                                state.tier1_advertise_proxies(&clock).await;
+                            }
+                        }
+                    });
+                }
+            }
+        });
```

Comment:
Do we need tested arbiter.spawn here? It seems the outer one can be just synchronous, no?